### ISSUE TITLE
docker build error

### ISSUE THREAD

**Issue by rgtlai** [NVIDIA/NeMo#3905]
**Describe the bug**
Error message during the docker build:
```
=> [internal] load build context                                                                                  0.7s
 => => transferring context: 34.81MB                                                                               0.6s
 => [nemo-src 1/1] COPY . .                                                                                        0.1s
 => [nemo-deps  2/14] RUN apt-get update &&     apt-get install -y     libsndfile1 sox     libfreetype6     pyth  59.1s
 => [nemo-deps  3/14] RUN pip uninstall -y sacrebleu torchtext                                                     0.9s
 => [nemo-deps  4/14] WORKDIR /tmp/torchaudio_build                                                                0.0s
 => ERROR [nemo-deps  5/14] RUN git clone --depth 1 --branch release/0.11 https://github.com/pytorch/audio.git &  12.4s
------
 > [nemo-deps  5/14] RUN git clone --depth 1 --branch release/0.11 https://github.com/pytorch/audio.git &&     cd audio &&     git submodule update --init --recursive &&     BUILD_SOX=1 python setup.py install &&     cd .. && rm -r audio:
#10 0.442 Cloning into 'audio'...
#10 1.963 Submodule 'kaldi' (https://github.com/kaldi-asr/kaldi) registered for path 'third_party/kaldi/submodule'
#10 1.964 Submodule 'third_party/kenlm/submodule' (https://github.com/kpu/kenlm) registered for path 'third_party/kenlm/submodule'
#10 1.975 Cloning into '/tmp/torchaudio_build/audio/third_party/kaldi/submodule'...
#10 9.030 Cloning into '/tmp/torchaudio_build/audio/third_party/kenlm/submodule'...
#10 10.54 Submodule path 'third_party/kaldi/submodule': checked out '3eea37dd09b55064e6362216f7e9a60641f29f09'
#10 10.66 Submodule path 'third_party/kenlm/submodule': checked out '5cea457db26950a73d638425c183b368c06ed7c6'
#10 12.13 -- Git branch: release/0.11
#10 12.13 -- Git SHA: 820b383b3b21fc06e91631a5b1e6ea1557836216
#10 12.13 -- Git tag: v0.11.0
#10 12.13 -- PyTorch dependency: torch==1.11.0a0+17540c5
#10 12.13 -- Building version 0.11.0+820b383
#10 12.13  --- Initializing submodules
#10 12.13  --- Initialized submodule
#10 12.13  --- Fetching zlib-1.2.11.tar.gz
#10 12.13 Traceback (most recent call last):
#10 12.13   File "setup.py", line 181, in <module>
#10 12.13     _main()
#10 12.13   File "setup.py", line 144, in _main
#10 12.13     _fetch_third_party_libraries()
#10 12.13   File "setup.py", line 128, in _fetch_third_party_libraries
#10 12.13     _fetch_archives(_parse_sources())
#10 12.13   File "setup.py", line 122, in _fetch_archives
#10 12.13     torch.hub.download_url_to_file(url, dest, progress=False)
#10 12.13   File "/opt/conda/lib/python3.8/site-packages/torch/hub.py", line 457, in download_url_to_file
#10 12.13     u = urlopen(req)
#10 12.13   File "/opt/conda/lib/python3.8/urllib/request.py", line 222, in urlopen
#10 12.13     return opener.open(url, data, timeout)
#10 12.13   File "/opt/conda/lib/python3.8/urllib/request.py", line 531, in open
#10 12.13     response = meth(req, response)
#10 12.13   File "/opt/conda/lib/python3.8/urllib/request.py", line 640, in http_response
#10 12.13     response = self.parent.error(
#10 12.13   File "/opt/conda/lib/python3.8/urllib/request.py", line 569, in error
#10 12.13     return self._call_chain(*args)
#10 12.13   File "/opt/conda/lib/python3.8/urllib/request.py", line 502, in _call_chain
#10 12.13     result = func(*args)
#10 12.13   File "/opt/conda/lib/python3.8/urllib/request.py", line 649, in http_error_default
#10 12.13     raise HTTPError(req.full_url, code, msg, hdrs, fp)
#10 12.13 urllib.error.HTTPError: HTTP Error 404: Not Found
------
executor failed running [/bin/sh -c git clone --depth 1 --branch release/0.11 https://github.com/pytorch/audio.git &&     cd audio &&     git submodule update --init --recursive &&     BUILD_SOX=1 python setup.py install &&     cd .. && rm -r audio]: exit code: 1
```

**Steps/Code to reproduce bug**
Run the command:
```
DOCKER_BUILDKIT=1 docker build -f Dockerfile -t nemo:latest .
```

**Expected behavior**
Docker image build without errors

**Environment overview (please complete the following information)**
Environment is on a Ubuntu 20.04 workstation with docker installed.

**Additional context**
GPU: A6000, CUDA 11.4

**Comment by coip** [comment]
fails given the `zlib-1.2.11` release torchaudio depends on was deleted for some reason, though `zlib-1.2.12` is fine to drop in.

can inject these following Dockerfile#L41 to get past it: 
``` bash
    sed -i 's|zlib.net/zlib-1.2.11.tar.gz|zlib.net/zlib-1.2.12.tar.gz|' third_party/zlib/CMakeLists.txt && \
    sed -i 's|c3e5e9fdd5004dcb542feda5ee4f0ff0744628baf8ed2dd5d66f8ca1197cb1a1|91844808532e5ce316b3c010929493c0244f3d37593afd6de04f71821d5136d9|' third_party/zlib/CMakeLists.txt && \
```

provenance:
https://github.com/pytorch/audio/issues/2295#issuecomment-1082257276

**Comment by mthrok** [comment]
Hi - We updated the zlib source to sourceforge.net, which is more reliable. The torchaudio build issue should be resolved now, but let us know if you encounter something else.

**Comment by rgtlai** [comment]
Was able to build the image so that is great. Found another issue relating the tutorials/nlp/Token_Classification-BioMegatron.ipynb notebook using the container from the image. While creating the model_ner it is trying to retrieve a model from Hugging Face.

ValueError: megatron-bert-cased is not supported by HuggingFace. megatron-bert-cased is not a local folder and is not a valid model identifier listed on 'https://huggingface.co/models'

I searched Hugging Face looks like the name of the model was renamed to megatron-bert-cased-345m ? I can create a separate ticket for this if you like as this is not related to the build issue.
